debian/raspberrypi-sys-mods.run_once.service

```diff
@@ -0,0 +1,18 @@
+# This unit is heavily inspired by Ned McClain's ideas found in https://github.com/nmcclain/raspberian-firstboot
+[Unit]
+Description=run_once initialization
+After=network.target
+Before=rc-local.service
+
+ConditionFileNotEmpty=/boot/run_once
+ConditionFileIsExecutable=yes
```

Comment:
Looking at https://www.freedesktop.org/software/systemd/man/systemd.unit.html , should this be `ConditionFileIsExecutable=/boot/run_once` ?

Reply:
indeed it should :)

Reply:
updated - I must have been sleeping :)